pkg/scyllaclient/client_scylla.go

```diff
@@ -940,11 +940,11 @@ func (c *Client) DisableAutoCompaction(ctx context.Context, keyspace, table stri
 }
 
 // FlushTable flushes writes stored in MemTable into SSTables stored on disk.
-func (c *Client) FlushTable(ctx context.Context, keyspace, table string) error {
+func (c *Client) FlushTable(ctx context.Context, host, keyspace, table string) error {
 	_, err := c.scyllaOps.StorageServiceKeyspaceFlushByKeyspacePost(&operations.StorageServiceKeyspaceFlushByKeyspacePostParams{
 		Cf:       &table,
 		Keyspace: keyspace,
-		Context:  ctx,
+		Context:  forceHost(ctx, host),
```

Comment:
I would suggest to move changes related to the tests in one commit, and fixes to the components in other, so that it would be easier to go over changes history in future.

Reply:
done